packages/pwa-kit-dev/src/configs/jest/jest.config.js

```diff
@@ -24,10 +24,7 @@ module.exports = {
     },
     globals: {
         DEBUG: true,
-        NODE_ENV: 'test',
-        Progressive: {
-            buildOrigin: '/mobify/bundle/development/'
-        }
+        NODE_ENV: 'test'
```

Comment:
Deleting the `/mobify/bundle/development/`here doesn't appear to change any test result. I don't think we're using it anywhere?

Reply:
I found what this is used for. It's used by getAssetUrl in template-retail-react-app tests. Will restore and come up with another way to have this be configurable.

Reply:
It doesn't need to be configurable. buildOrigin just needs to be any non-empty string. It is used by the tests to mock an asset url.

I restored the string as '/mobify/bundle/development/' and left a comment on what this property is used for.

It does not need to be configurable via namespacing.